src/store/sagas/notificationsSaga.ts

```diff
@@ -0,0 +1,17 @@
+import { takeEvery } from 'redux-saga/effects';
+import { productActionTypes } from '../actionTypes/productActionTypes';
+import { notification } from 'antd';
+import notifications from '../../resources/notifications';
+
+function* successProductCreation() {
+  yield notification.success(notifications.success);
+}
```

Comment:
`function* successProductCreation(action) {
  yield notification.success(action.message);
}`

the message text should be passed from action